sormas-e2e-tests/src/main/java/org/sormas/e2etests/pages/application/events/EventDirectoryPage.java

```diff
@@ -37,6 +37,7 @@
       By.cssSelector("div.v-grid-tablewrapper tbody tr");
   public static final By EVENT_ACTIONS_TABLE_DATA = By.tagName("td");
   public static final By FIRST_EVENT_ID_BUTTON = By.cssSelector(".v-grid-row-has-data a[title]");
+  public static final By CREATE_CASE_BUTTON= By.xpath("//a[@title='Create new case']");
```

Comment:
I know that this approach is common among the project, but we can improve it a bit by adding other identifier, because if we'll start using the tests on different markets, the language will be different and will fail